editquality/utilities/autolabel.py

```diff
@@ -124,22 +125,23 @@ def run(api_host, revisions, labels_f, trusted_groups, trusted_edits,
                             revert_radius, revert_window, exclude_reverted,
                             exclude_reverting)
 
+    revisions, revisions2 = tee(revisions)
+    number_of_revisions = sum(1 for line in revisions2)
```

Comment:
https://docs.python.org/3/library/itertools.html#itertools.tee

> This itertool may require significant auxiliary storage (depending on how much temporary data needs to be stored). In general, if one iterator uses most or all of the data before another iterator starts, it is faster to use list() instead of tee().

Maybe we should have just read in all of the revisions into a list and run len() on that.  Either way, this is plenty fast as is.